actors/miner/tests/declare_recoveries.rs

```diff
@@ -0,0 +1,174 @@
+use fil_actor_miner::pledge_penalty_for_continued_fault;
+use fil_actor_miner::power_for_sectors;
+use fil_actors_runtime::test_utils::expect_abort_contains_message;
+use fvm_ipld_bitfield::BitField;
+use fvm_shared::address::Address;
+use fvm_shared::bigint::BigInt;
+use fvm_shared::clock::ChainEpoch;
+use fvm_shared::consensus::{ConsensusFault, ConsensusFaultType};
+use fvm_shared::econ::TokenAmount;
+use fvm_shared::error::ExitCode;
+
+mod util;
+use crate::util::*;
+
+// an expriration ~10 days greater than effective min expiration taking into account 30 days max
+// between pre and prove commit
+const DEFAULT_SECTOR_EXPIRATION: ChainEpoch = 220;
```

Comment:
you can use the one defined [here](https://github.com/filecoin-project/builtin-actors/blob/a4a4c869ff1f5af0ce99d4f7e76116ca24efc8c0/actors/miner/tests/util.rs#L83)